app/vmbackup/main.go

```diff
@@ -92,7 +92,9 @@ func main() {
 	}
 	srcFS.MustStop()
 	dstFS.MustStop()
-	originFS.MustStop()
+	if originFS != nil {
```

Comment:
probably, its better to return &fsnil.FS{} from newOriginFS constructor? as far as i see, its later replaced with it at https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/lib/backup/actions/backup.go#L53